base/src/ImageViewerModule.cpp

```diff
@@ -8,13 +8,73 @@
 #include "Logger.h"
 #include "Utils.h"
 
-class ImageViewerModule::Detail
+#if defined(__arm__) || defined(__aarch64__)
```

Comment:
Put a comment describing this:
arm : EGL Renderer
win/linux: imshow

Reply:
added a comment